libs/standard-tests/langchain_standard_tests/integration_tests/chat_models.py

```diff
@@ -133,7 +133,7 @@ def test_conversation(self, model: BaseChatModel) -> None:
         assert isinstance(result.content, str)
         assert len(result.content) > 0
 
-    def test_usage_metadata(self, model: BaseChatModel) -> None:
+    async def test_usage_metadata(self, model: BaseChatModel) -> None:
```

Comment:
Could you duplicate the sync test and create an async version of it? Instead of doing both sync and async in the same test?